renderer/renderer.py

```diff
@@ -36,26 +36,7 @@ def __init__(self,  config_filename, layout=[], tiling=None, players=[], window_
         self.window_size = window_size
         # The framerate of the renderer. This isn't too important since the renderer
         # displays static drawings.
-        self.render_fps = render_fps
-        # The pygame window
-        self.window = None
-        # The pygame clock
-        self.clock = None
-    
-    def _init_setup(self, render_mode):
-        """
-        This function initializes the pygame window and clock if not already initialized.
-
-        Args:
-            render_mode (str): Either "human" or "rgb_array"
-        """
-        if self.window is None and render_mode == "human":
-            pygame.init()
-            pygame.display.init()
-            self.window = pygame.display.set_mode(self.window_size)
-            pygame.display.set_caption('Robotouille Simulator')
-        if self.clock is None and render_mode == "human":
-            self.clock = pygame.time.Clock()
+        pygame.display.set_mode(self.window_size)
```

Comment:
This is not necessary. I left this piece of code because if the window size is not 512 x 512, the game does not run correctly. In addition, clicking on the black borders causes the game to crash. If you want to remove this code, there are a few options: (1) making all the screens be 512 x 512 for now or (2) I can center the screen with black borders, but it will crash if you click outside the game region.

Reply:
I'd put a TODO in the code for now so we can correct this later. Ideally we can center the game screen, make the screen as large as can fit in the screen, and write code to prevent crashing by clicking the game region.